build.py

```diff
@@ -282,30 +287,38 @@ def print_usage():
             sys.argv[0]
     sys.exit(2)
 
+def update_benchmark_path(path):
+    print 'Specified benchmark path %s' % path
+    cxx_flags.extend(['-I', '%s/include'%(path)])
+
+
 def main():
     os.chdir(BESS_DIR)
+    parser = argparse.ArgumentParser(description = 'Build Bess')
+    parser.add_argument('action', metavar='action', nargs='?', default='all', \
+            help='Action is one of all, dpdk, bess, kmod, clean, dist_clean, help')
+    parser.add_argument('--with-benchmark', dest='benchmark_path', nargs=1, help='Location of benchmark library')
```

Comment:
Rather than introducing new options, one may just run `CXXFLAGS=-I/opt/benchmark/include build.py` on the shell? (or `export` the flag so that it can persist for subsequent builds in the `core` directory)